cfme/tests/intelligence/test_reports_with_timelines.py

```diff
@@ -5,22 +5,47 @@
 
 
 @pytest.mark.manual
+@pytest.mark.tier(2)
 @pytest.mark.parametrize('report', timelines_reports)
 def test_default_reports_with_timelines(report):
     """
+    Test that a timeline widget is displayed for the default reports.
+    Note that since the default reports look at last week, to test, we modify the reports
+
     Polarion:
-        assignee: None
-        initialEstimate: None
+        assignee: jdupuy
+        initialEstimate: 1/5h
+        casecomponent: web_ui
+        caseimportance: medium
+        testSteps:
+            1. Add an infrastructure provider to ensure that an event for each report will occur
+            2. Navigate to Cloud Intel -> Reports
+            3. Modify one of the default reports to display events from today
+            4. Navigate to Cloud Intel -> Timelines
+            5. Click on the report you created
+            6. Verify that the timeline (with events) is properly displayed
     """
     pass
 
 
 @pytest.mark.manual
+@pytest.mark.tier(2)
 @pytest.mark.parametrize('report', timelines_reports)
 def test_custom_reports_with_timelines(report):
     """
+    Test that a timeline widget is displayed for custom reports.
+
     Polarion:
-        assignee: None
-        initialEstimate: None
+        assignee: pvala
```

Comment:
Question:
why you are moving her test? Is this test belong to her?

Reply:
the tests removed from `test_manual.py` are indeed duplicates -- these manual cases already existed in `integration_tests`. The two remaining tests are actually 8 tests since they are parametrized against the `timelines_reports`.

Reply:
@digitronik regarding pvala's test, she is marked as the assignee on Polarion, but I will reach out to her to see if this is not the case.

Reply:
@digitronik after reaching out to @valaparthvi it looks like the test case is not her's. I have assigned it to myself since it has to do with events and event timelines.